pages/index.tsx

```diff
@@ -38,52 +50,49 @@ const Home: NextPage = () => {
-							/>
-							<path
-								d="M32.09 55.54c3.29 7.84 12.81 14.46 19 18.74 1.17.81 2.18 1.52 3.05 2.16a1.67 1.67 0 0 0 1 .33 1.62 1.62 0 0 0 1-.33c.88-.64 1.89-1.35 3-2.16 6.15-4.27 15.68-10.9 19-18.74Z"
-								style={{
-									fill: "#ffd500",
-								}}
-							/>
-						</svg>
+			<div className="max-w-7xl mx-auto pt-1 px-3 md:px-8 text-xs md:text-base leading-normal">
+				<div className="grid gap-x-4 gap-y-3 grid-cols-1 md:grid-cols-2">
+					<div className={cx(styles.card)}>
```

Comment:
neni treba pouzivat `classnames` utilitu, ta se hodi jen kdyz je treba poskladat nekolik util trid na zaklade nejakych podminek, tam, kde se aplikuje jen jeden styl z module.css je to zbytecne

Reply:
ja jsem chtela na 2 mistech pouzit stejne classNames, tak jsem to resila takto.. ale pokud to je zbytecne, tak to vecer predelam